apiClient.go

```diff
@@ -32,51 +34,100 @@ type apiClient struct {
 	apiToken   string
 }
 
-func (c *apiClient) GetStatus(ctx context.Context, repoSource, repoOwner, repoName string) (status string, err error) {
+func (c *apiClient) OrganizationProjects(ctx context.Context, orgID string) (Projects, error) {
 
-	getStatusURL := fmt.Sprintf("%v/...", c.apiBaseURL)
+	var wrapper struct {
+		Projects `json:"projects"`
+	}
+
+	organizationProjectsURL := fmt.Sprintf("%v/org/%s/projects", c.apiBaseURL, orgID)
 
 	headers := map[string]string{
-		"Content-Type": "application/json",
+		"Authorization": fmt.Sprintf("token %s", c.apiToken),
+		"Content-Type":  "application/json",
 	}
 
-	responseBody, err := c.getRequest(getStatusURL, nil, headers)
+	response, err := c.getRequest(organizationProjectsURL, nil, headers, 200)
 	if err != nil {
-		log.Error().Err(err).Str("url", getStatusURL).Msgf("Failed retrieving snyk status")
-		return
+		log.Fatal().Msgf("Failed retrieving projects from snyk")
 	}
 
-	var statusResponse struct {
-		Status string `json:"status"`
+	defer response.Body.Close()
+
+	bodyBytes, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		log.Info().Msgf(err.Error())
+		log.Fatal().Msgf("Error reading body of snyk response")
```

Comment:
No need for the separate Info log, you can add `.Err(err)` to the Fatal log and have the error rendered as well.